lib/modules/apostrophe-schemas/public/js/user.js

```diff
@@ -1040,6 +1039,41 @@ apos.define('apostrophe-schemas', {
       convert: convertString
     });
 
+    self.addFieldType({
+      name: 'colorpicker',
+      populate: function (data, name, $field, $el, field, callback) {
+        $field.val(data[name]);
+        apos.ui.enhanceColorpicker($field);
```

Comment:
This kinda makes me wish the whole field lived in its own module, like apostrophe-attachments, but I could go either way on that. We have a lot of modules and it's not as big a deal as apostrophe-attachments.